app/pb_kits/playbook/pb_stat_change/stat_change.rb

```diff
@@ -46,47 +33,26 @@ def icon
         end
       end
 
-      def display_icon
-        if !icon.nil?
-          pb_icon = Playbook::PbIcon::Icon.new(icon: icon, fixed_width: true)
-          ApplicationController.renderer.render(partial: pb_icon, as: :object)
-        else
-          ""
-        end
-      end
-
-      def value
-        default_value(configured_value, "")
-      end
-
-      def display_value
-        pb_icon_element = Playbook::PbBody::Body.new(status: status) do
-          display_icon +
-            value
-        end
-        ApplicationController.renderer.render(partial: pb_icon_element, as: :object)
-      end
-
-      def kit_class
-        stat_options = [
-          "pb_stat_change_kit",
-          status,
-        ]
-        stat_options.join("_")
-      end
-
-      def to_partial_path
-        "pb_stat_change/stat_change"
-      end
+      # def display_icon
+      #   if !icon.nil?
+      #     pb_icon = Playbook::PbIcon::Icon.new(icon: icon, fixed_width: true)
+      #     ApplicationController.renderer.render(partial: pb_icon, as: :object)
+      #   else
+      #     ""
+      #   end
+      # end
```

Comment:
Let's remove this if it's not being used.

Reply:
i removed it! I was still working on it at the time and still had my commented out code to guide me!